style-guide/src/components/section-header/styles.module.css

```diff
@@ -1,11 +1,12 @@
-@value spacing-large, spacing-medium, large-bold-font-size, large-bold-font-weight, large-bold-line-height, color-gray-primary, color-gray-utility from '../../styles/tokens.module.css';
+@value spacing-large, spacing-medium, large-bold-font-size, large-bold-font-weight, large-bold-line-height from '../../styles/tokens.module.css';
```

Comment:
I would actually prefer to remove `tokens.module.css` and directly reference the token values from the NPM module names, but I decided to try and keep the diffs small.

Reply:
yeah this was an hack we did because of a gatsby issue, if I remember it uses an old version of the webpack extract plugin which doesn't support css modules to be compiled with ignoreOrder: true option and so this was enough for the styleguide to build without errors.
If now gatsby moved to webpack 3 or 4, perhaps the issue is gone and we can do this ;)

Reply:
For your reference, issue and PR: https://github.com/opentable/design-tokens/issues/134

Reply:
Oh, I didn't know! Hm. TIL. I did run `npm run build` in `styleguide` to do the build, but I didn't inspect the css nor verify the static site looked correct. It didn't fail to build, though. Let me check that tomorrow.

Reply:
yeah, I mean this should be fine as you said in case it would be in a separate PR ;)